storage/tianmu/handler/tianmu_handler.cpp

```diff
@@ -544,10 +544,7 @@ int TianmuHandler::delete_row([[maybe_unused]] const uchar *buf) {
  Called from sql_select.cc by JOIN::rein*it.
  Called from sql_union.cc by st_select_lex_unit::exec().
  */
-int TianmuHandler::delete_all_rows() {
```

Comment:
what will be happened under multi-sessions?

Reply:
In the SQL layer, the tables to be operated have been locked to support users' synchronous and mutually exclusive access. There is no need to perform parallel access control again in the engine.